### ISSUE TITLE
vomnibar's font changed after the update on Firefox

### ISSUE THREAD

**Issue by ngntrgduc** [philc/vimium#4350]
**Describe the bug**

After I updated Vimium, it seemed like the vomnibar's font was changed on Firefox:

![image](https://github.com/philc/vimium/assets/47920109/494cb1c8-ab81-41f2-af06-67af7742a2bc)

But it's still normal on Edge:

![image](https://github.com/philc/vimium/assets/47920109/8b267341-87ee-48b1-a3a1-89e73db6a633)

**To Reproduce**

1. Install Vimium for Firefox: https://addons.mozilla.org/en-US/firefox/addon/vimium-ff/
2. Open vomnibar and start typing

**Browser and Vimium version**

- Firefox version: `119.0`
- Vimium version: `2.0.4`
- Edge version: `118.0.2088.69 (Official build) (64-bit) `

**Comment by philc** [comment]
I'm not able to reproduce this on Mac. What is your operating system, and does it happen on all URLs?

**Comment by ngntrgduc** [comment]
My OS is Windows.

Yes, it happens on all URLs.

**Comment by philc** [comment]
I wasn't able to reproduce this on Windows 11 ARM with Firefox 119; see screenshot below. I don't see why the fonts would look different on Firefox vs. Edge, which is conspicuous. However, notice in your screenshot that your input bar in Firefox uses a sans serif font, whereas the rest of the dialog doesn't. I looked at the CSS, and the whole Vomnibar dialog has "sans-serif" as its font, so it's possible that somehow your Firefox isn't resolving this to a sans serif font. The input box in the Vomnibar is different because it has more options in its font-family: Helvetica, Arial, sans-serif.

In any case, they dialog vs. the input box shouldn't have different font families, so I've combined them in 3d25292b3605bd752594ff611c099f11164239c6, which may fix this issue.
